src/main/java/teammates/ui/controller/AdminEmailComposeSendAction.java

```diff
@@ -145,7 +135,7 @@ private void moveJobToAddressModeTaskQueue() {
         }
         taskQueuer.scheduleAdminEmailPreparationInAddressMode(emailId, addressReceiverListString);
 
-        statusToAdmin = "Email queued for sending." + "<br/>" + "Receipient: " + addressReceiverListString;
+        statusToAdmin += "<br/>" + "Receipient: " + addressReceiverListString;
```

Comment:
* Receipient -> recipient?
* Might not be safe to do a +=. @xpddavid correct me if I'm wrong, but don't think statusToAdmin is default initialized with an empty String in Action; its just declared as String statusToAdmin without being initialized before Action.execute()